### PR TITLE
password inputs disappear and get reset on account page

### PR DESCRIPTION
**Pull request by compassface**
#255

### CODE REVIEW

**Comment by joshuarichard** [comment]
found lint :(

**Comment by joshuarichard** [comment]
found lint :(

**Comment by joshuarichard** [comment]
success! no lint!

File: public/js/account.js [hunk 1]
```diff
@@ -435,6 +435,16 @@ $(document).ready(function() {
                                     },
                                     success: function() {
                                         alert('Password successful changed.');
+                                        $('#form-old-password').val('');
+                                        $('#form-password').val('');
+                                        $('#form-confirm-password').val('');
+                                        $('#form-first-name').prop('disabled', false);
+                                        $('#form-last-name').prop('disabled', false);
```
Comment (joshuarichard) [inline public/js/account.js:444]: @jacobbuettner were we doing 'disabled', false or .hide() and .show()?

**Comment by compassface** [comment]
so what disabled is doing is graying out the text boxes, not hiding them or showing them. I have to set those to false after success in order for the donor to change their other information

File: public/js/account.js (hunk 1 above)
Comment (jacobbuettner) [inline public/js/account.js:444]: it doesn't matter either way, I think using .hide() and .show() is a little easier and clearer so I recommend we use that.  I changed the line to use .hide()

**Pull request merged**